gnome-extra/gnome-commander/Manifest

```diff
@@ -1,4 +1,4 @@
-DIST gnome-commander-1.6.1.tar.xz 6550308 BLAKE2B d4353bce913a43e091f2d66b16c932ff8c3d863a0dd323f487a8ef4b9ffacbe4e62e70600d4fb8e5ca15759c3c07888a17662e328abbbb7013446d294986c11d SHA512 717e8bb560c6c7e6505cfee1d9ab12f7b3d5c2f8131b9eaf8b15e13aa71dad85574b14ae6f3a42dae42d8a6a291d33cd71db5a4155184bec4f66e3db8b2daab0
```

Comment:
Please split this into two commits, one adding the new version, and the other removing old.

Reply:
Okay, will do that. Shall I open a bug report for the removal of version 1.6.1?

Reply:
No. Bug reports are not necessary for the packages you're maintaining.